src/_includes/layout.njk

```diff
@@ -59,7 +59,7 @@
       <div class="container">
         <a class="navbar-brand js-scroll-trigger" href="#page-top">Code Connector</a>
         <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
-          <span class="d-none d-md-inline">Menu</span><span class="d-inline d-md-none"><i class="fas fa-bars"></i></span>
+          <span class="d-none d-md-inline">Menu</span><span class="d-inline d-md-none" aria-label="Menu Button"><i class="fas fa-bars"></i></span>
```

Comment:
I don't think this was needed. The button has aria attributes. What triggered you to add this?

Reply:
In fact, I think a screen reader will ignore the span, because the button has the aria-label.

Reply:
I removed